AlphaWallet/TokenScriptClient/Coordinators/EventSourceCoordinator.swift

```diff
@@ -39,6 +39,14 @@ final class EventSourceCoordinator: NSObject {
     private func subscribeForTokenChanges() {
         tokensDataStore
             .enabledTokensChangeset(for: enabledServers)
+            .map { changeset -> [Token] in
+              switch changeset {
+              case .initial(let tokens): return tokens
+              case .update(let tokens, _, _, _): return tokens
+              case .error: return []
+              }
+            }
+            .removeDuplicates()
```

Comment:
yes, it could be, `removeDuplicates` filters same results that get filred in source publisher
```
send([a, b, c, a])
send([a, b, c, a])
send([a, b, c, a])
....
sink([a, b, c, a])
```

Reply:
so there could be same elements, in received event